### ISSUE TITLE
For the dataset GenomicBenchmarks

### ISSUE THREAD

**Issue by Eyulaochen** [HazyResearch/hyena-dna#80]
Dear Sir,

Could you please tell me which model is used for GenomicBenchmarks.
'hyenadna-tiny-1k-seqlen' or..
What is the learning rate for this datasets.

Many thanks!

Best regards,
Huayang

**Comment by Eyulaochen** [comment]
What's the hyperparameters for GenomicBenchmarks?

**Comment by kps333** [comment]
You may open the config files to see the pre-set parameters
